charts/seed-bootstrap/templates/grafana/grafana-deployment.yaml

```diff
@@ -37,6 +37,8 @@ spec:
           value: "false"
         - name: GF_AUTH_ANONYMOUS_ENABLED
           value: "true"
+        - name: GF_USERS_VIEWERS_CAN_EDIT
```

Comment:
IIUC, the idea of this PR is to prevent the login in Grafana with the default username and password (`admin` and `admin`). If this is the case, I guess the corresponding docs needs an update as well - ref https://github.com/gardener/gardener/blob/master/docs/usage/logging.md#how-to-access-the-logs.

Reply:
Good point, done :)